runtime/ajuna/src/xcm_config.rs

```diff
@@ -322,13 +322,33 @@ parameter_types! {
 	pub RelayNativePerSecond: (AssetId, u128,u128) = (Location::new(1,Here).into(), AJUN * 70, 0u128);
 	// Weight for one XCM operation.
 	pub UnitWeightCost: Weight = Weight::from_parts(1_000_000u64, DEFAULT_PROOF_SIZE);
+
 	pub const AjunNative: AssetFilter = Wild(AllOf { fun: WildFungible, id: AssetId(Location::here()) });
 	pub AssetHubTrustedTeleporter: (AssetFilter, Location) = (AjunNative::get(), AssetHubLocation::get());
+
+	pub const RelayLocation: Location = Location::parent();
+	pub RelayLocationFilter: AssetFilter = Wild(AllOf {
+		fun: WildFungible,
+		id: AssetId(RelayLocation::get()),
+	});
+
+	/// Dot from Asset Hub
+	pub RelayChainNativeAssetFromAssetHub: (AssetFilter, Location) = (
+		RelayLocationFilter::get(),
+		AssetHubLocation::get()
+	);
 }
 
-pub type TrustedTeleporters = (staging_xcm_builder::Case<AssetHubTrustedTeleporter>,);
+pub type TrustedTeleporters = (Case<AssetHubTrustedTeleporter>,);
 
-pub type Reserves = (NativeAsset, ReserveAssetsFrom<AssetHubLocation>);
```

Comment:
Before, we allowed all assets, for which the XCM origin was the same as the assets location. This is very permissive, and we should prefer to make it explicit.